Rename driver_init to fcdslsl_driver_init, as it clashes with
driver_init defined in <linux/driver.h> header since 2.6.20. While at it
renamed driver_exit to fcdslsl_driver_exit, not really needed but did it
as a cosmetic thing and to avoid the same problem in the future like the
driver_init case.

Signed-off-by: Herton Ronaldo Krzesinski <herton@mandriva.com>

--- fritz/src.orig/driver.c	2008-01-05 23:39:51.000000000 -0200
+++ fritz/src/driver.c	2008-01-05 23:52:14.000000000 -0200
@@ -1563,19 +1563,19 @@ void __stack init (unsigned len, void (_
 
 /*---------------------------------------------------------------------------*\
 \*---------------------------------------------------------------------------*/
-int driver_init (void) {
+int fcdslsl_driver_init (void) {
 
 	return (NULL != (capi_lib = link_library (&capi_card)));
-} /* driver_init */
+} /* fcdslsl_driver_init */
 
 /*---------------------------------------------------------------------------*\
 \*---------------------------------------------------------------------------*/
-void driver_exit (void) {
+void fcdslsl_driver_exit (void) {
 
 	assert (capi_lib);
 	free_library ();
 	capi_lib = NULL;
-} /* driver_exit */
+} /* fcdslsl_driver_exit */
 
 /*---------------------------------------------------------------------------*\
 \*---------------------------------------------------------------------------*/
--- fritz/src.orig/driver.h	2008-01-05 23:41:51.000000000 -0200
+++ fritz/src/driver.h	2008-01-05 23:52:31.000000000 -0200
@@ -161,8 +161,8 @@ extern void init (unsigned, void (__attr
 
 /*---------------------------------------------------------------------------*\
 \*---------------------------------------------------------------------------*/
-extern int driver_init (void);
-extern void driver_exit (void);
+extern int fcdslsl_driver_init (void);
+extern void fcdslsl_driver_exit (void);
 
 #endif
 
--- fritz/src.orig/main.c	2008-01-05 23:42:04.000000000 -0200
+++ fritz/src/main.c	2008-01-05 23:53:19.000000000 -0200
@@ -116,14 +116,14 @@ static int __devinit fcdsl2_probe (
 		return -ENODEV;
 	}
 	NOTE("Loading...\n");
-	if (!driver_init ()) {
+	if (!fcdslsl_driver_init ()) {
 		ERROR("Error: Driver library not available.\n");
 		NOTE("Not loaded.\n");
 		return -ENOSYS;
 	}
 	if (0 != (res = add_card (dev))) {
 		NOTE("Not loaded.\n");
-		driver_exit ();
+		fcdslsl_driver_exit ();
 		return res;
 	}
 	NOTE("Loaded.\n");
@@ -144,7 +144,7 @@ static void __devexit fcdsl2_remove (str
 	NOTE("Removing...\n");
 	remove_ctrls (cp);
 	NOTE("Removed.\n");
-	driver_exit ();
+	fcdslsl_driver_exit ();
 #ifndef NDEBUG
 	if (hallocated() != 0) {
 		ERROR("%u bytes leaked.\n", hallocated());
